tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+extern gpio_t DASHBOARD_POWER;
+extern gpio_t SERVICE_SECTION_POWER;
+extern gpio_t THROTTLE_POWER;
+
+void init_devices();
+void power_device();
```

Comment:
make sure you're adding in arguments for the functions that have them! if there are no arguments, you can put void